ci-operator/config/rhoar-qe/spring-boot-openshift-interop-tests/rhoar-qe-spring-boot-openshift-interop-tests-main__springboot-ocp4.17-lp-interop.yaml

```diff
@@ -49,6 +49,32 @@ tests:
     - ref: springboot-prepare-cluster
     - ref: springboot-tests-interop
     workflow: firewatch-ipi-aws
+- as: springboot-interop-aws-fips
+  cron: 0 6 11 2 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      FIPS_ENABLED: "true"
+      FIREWATCH_CONFIG: |
+        {
+          "failure_rules":
+            [
+              {"step": "springboot-prepare-cluster", "failure_type": "all", "classification":  "Infrastructure Provisioning - Cluster Preparation", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default","interop-tests"]},
+              {"step": "springboot-tests-interop", "failure_type": "pod_failure", "classification": "Test Execution", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["!default","interop-tests"]},
+              {"step": "springboot-tests-interop", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "ENTSBT", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["!default","interop-tests"]}
+            ]
+        }
+      FIREWATCH_CONFIG_FILE_PATH: https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/aws-ipi/lp-interop.json
+      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.17-lp","self-managed-lp","springboot-lp"]'
```

Comment:
@amp-rh - Requires `"fips"` in `FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS` array.